src/include/sof/common.h

```diff
@@ -12,7 +12,7 @@
 /* Align the number to the nearest alignment value */
 #define IS_ALIGNED(size, alignment) ((size) % (alignment) == 0)
 #define ALIGN_UP(size, alignment) \
-	((size) + (((alignment) - ((size) % (alignment))) % (alignment)))
+	((size) + (alignment) - 1 - ((size) + (alignment) - 1) % (alignment))
```

Comment:
Can't we in the future impose that alignment must be a power of 2, and use bitwise operations to avoid divisions altogether? When would we in practice need an alignment that is _not_ a power of 2 anyway?

Reply:
@paulstelian97 Yes, I want to do that, but for now I'm trying to find out why these patches fail CI...

Reply:
I think that now the CI simply doesn't launch until explicitly requested or something? I remembered @xiulipan did a few things yesterday that might cause this, for I cannot find any potential issue with this patch specifically.

Reply:
@lyakh CI is looking good for this one now.